rumbl/web/controllers/auth.ex

```diff
@@ -8,22 +8,33 @@ defmodule Rumbl.Auth do
 
   def call(conn, repo) do
     user_id = get_session(conn, :user_id)
+
     cond do
       user = conn.assigns[:current_user] -> conn
+        put_current_user(conn, user)
```

Comment:
I'm not saying to pipe it.

I'm saying the `conn` on the end of line 12/13 is not doing anything.